app/components/side-menu/template.hbs

```diff
@@ -0,0 +1,31 @@
+<div class="side-menu">
+    <form action="#" id="search-form">
+        <input type="text" placeholder="&#xF002;" style="font-family:Arial, FontAwesome">
+    </form>
+
+    <div class="side-menu-filters">
+        <p>FILTER BY</p>
+        <div class="side-menu-item active">
+            <a href="#">All</a>
+        </div>
+        <div class="side-menu-item">
+            <a href="#">Full Time</a>
```

Comment:
The naming here will either need to be dynamic based on the page its being rendered in or we need to rename this from side menu to something more specific to jobs page

Reply:
to pull in data, you will use route.hbs file and `Model()` method...
```
 model() {
    return this.store.findAll('photos');
  }
```
 once you do that, then you can simply pull that data into template.hbs file by doing something similar to 
```<ul>
  {{#each this.photos as |p|}}
    <li>
      <LinkTo @route="photos.edit" @model={{p}}>{{p.title}}</LinkTo>
    </li>
  {{/each}}
</ul>

Reply:
Wouldn't each page have a side-menu and wouldn't the styling be the same? Sure, different parts of the side menu could be rendered and with different data but seems like the class names in this component would just be for styling.